scripts/export.js

```diff
@@ -44,7 +44,14 @@ $(document).ready(() => {
         if (document.getElementById('feature-check').checked) {
             options.push('feature=' + $("#feature-select option:selected")[0].value);
         }
-        window.location.href = '/export?' + options.join('&');
+        let exportRef = '/export?' + options.join('&');
+        $.get(exportRef, downloadFile => {
```

Comment:
this is fine for the conference, but this can't be our long term solution. It gets the file twice, which means the server processes each request twice every time